lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -786,155 +774,155 @@ instance FromBlockfrost BF.Block BlockHeader where
-        maxBlockHeaderSize <-
-            _protocolParamsMaxBlockHeaderSize <?#> "MaxBlockHeaderSize"
-        maxBlockBodySize <-
-            _protocolParamsMaxBlockSize <?#> "MaxBlockBodySize"
-        eMax <-
-            _protocolParamsEMax <?#> "EMax"
-        nOpt <-
-            _protocolParamsNOpt <?#> "NOpt"
-
-        pure
-            ProtocolParameters
-                { eras = emptyEraInfo
-                , txParameters =
-                    TxParameters
-                        { getFeePolicy =
-                            LinearFee $
-                                LinearFunction
-                                    { intercept = fromIntegral minFeeB
-                                    , slope = fromIntegral minFeeA
-                                    }
-                        , getTxMaxSize =
-                            Quantity maxTxSize
-                        , getTokenBundleMaxSize =
-                            TokenBundleMaxSize $ TxSize maxValSize
-                        , getMaxExecutionUnits =
-                            ExecutionUnits
+                    , protocolParamExtraPraosEntropy = Nothing
+                    , protocolParamMaxBlockHeaderSize = maxBlockHeaderSize
+                    , protocolParamMaxBlockBodySize = maxBlockBodySize
+                    , protocolParamMaxTxSize = intCast maxTxSize
+                    , protocolParamTxFeeFixed = minFeeB
+                    , protocolParamTxFeePerByte = minFeeA
+                    , protocolParamMinUTxOValue =
+                        Just $ Node.Lovelace $ intCast _protocolParamsMinUtxo
+                    , protocolParamStakeAddressDeposit =
+                        Node.Lovelace $
+                            intCast @_ @Integer _protocolParamsKeyDeposit
+                    , protocolParamStakePoolDeposit =
+                        Node.Lovelace $
+                            intCast @_ @Integer _protocolParamsPoolDeposit
+                    , protocolParamMinPoolCost =
+                        Node.Lovelace $
+                            intCast @_ @Integer _protocolParamsMinPoolCost
+                    , protocolParamPoolRetireMaxEpoch = Node.EpochNo eMax
+                    , protocolParamStakePoolTargetNum = nOpt
+                    , protocolParamPoolPledgeInfluence =
+                        toRational _protocolParamsA0
+                    , protocolParamMonetaryExpansion = toRational _protocolParamsRho
+                    , protocolParamTreasuryCut = toRational _protocolParamsTau
+                    , protocolParamUTxOCostPerWord =
+                        Just $
+                            Node.Lovelace $
+                                intCast _protocolParamsCoinsPerUtxoWord
+                    , protocolParamCostModels =
+                        Map.singleton
+                            (AnyPlutusScriptVersion PlutusScriptV1)
+                            (Fixture.costModels network)
```

Comment:
👍

For the hard-fork, we'll need to add a cost model for `PlutusScriptV2` as well. Not sure if it's publicly available yet. Could you make a ticket so we do not forget?